src/Chapter1.hs

```diff
@@ -641,7 +670,8 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n = if (abs n) < 10 then (abs n) else (firstDigit (div (abs n) 10))
```

Comment:
That is absolutely correct 💯 

Note, that you can omit parenthesis in all cases of the if-then-else. You can write it like:
```suggestion
firstDigit n = if abs n < 10 then abs n else firstDigit (div (abs n) 10)
```

You can see that `abs n` is reused several times in this function. This usually means that you can move it into the `where` case and reuse everywhere 👍🏼